scripts/deno/ophan-components.ts

```diff
@@ -64,9 +63,9 @@ const getOphanComponents = (
 	const doc = new DOMParser().parseFromString(html, 'text/html');
 	if (!doc) throw new Error('Unable to parse DOM');
 
-	return [...doc.querySelectorAll(`[${attribute}]:not(.is-hidden,.is-hidden *)`)].filter(
-		(node): node is Element => node instanceof Element,
-	);
+	return [
+		...doc.querySelectorAll(`[${attribute}]:not(.is-hidden,.is-hidden *)`),
+	].filter((node): node is Element => node instanceof Element);
```

Comment:
is this meant to be in here too?

```suggestion
	return [...doc.querySelectorAll(`[${attribute}]:not(.is-hidden,.is-hidden *)`)].filter(
		(node): node is Element => node instanceof Element,
	);
```